src/datascience-ui/ipywidgets/kernel/index.ts

```diff
@@ -114,15 +114,6 @@ export function disposeOutput(e: { outputId: string } | undefined) {
         );
     }
 }
-function renderErrorInLastOutputThatHasNotRendered(message: string) {
-    const possiblyEmptyOutputElement = [...stackOfWidgetsRenderStatusByOutputId]
-        .reverse()
-        .find((item) => !item.success);
-    if (possiblyEmptyOutputElement) {
-        //
-        console.log(message);
```

Comment:
This is what was missing, I'm now display the message as a messagebox to the user.

Reply:
FYI @rchiodo I was getting the prompt in Native Notebooks.
Had to find what was missing, & the only missing part was this area.
Lets see when this gets validated.